src/contracts/Hub.test.ts

```diff
@@ -0,0 +1,56 @@
+import { Eth } from 'web3-eth';
+import { HttpProvider } from 'web3-providers';
+import { Environment } from '../Environment';
+import deployment from '../deployments/mainnet';
+import { Hub, FundRoutes } from './Hub';
+import { toChecksumAddress } from 'web3-utils';
+
+describe('Hub', () => {
+  let environment: Environment;
+  let hub: Hub;
+  let routes: FundRoutes;
+
+  beforeAll(() => {
+    // TODO: This should be replaced with a local ganache test environment using proper test fixtures.
+    const client = new Eth(new HttpProvider('https://mainnet.melonport.com'));
+    environment = new Environment(client, deployment);
+    hub = new Hub(environment, '0x7362d4dd194cf5174748317da1e960723b78f28c');
+  });
+
+  it('should return the name as "miFund"', async () => {
+    const result = await hub.name();
+    expect(result).toBe('miFund');
```

Comment:
Same here. Not sure if toBe() or toEqual() are correct for this.

Reply:
For primitive types, it's doesn't make a difference, apparently, see https://stackoverflow.com/a/27929503

Reply:
Okay perfect then.